app/src/main/java/com/toptier/targetmvd/ui/views/SignUpView.java

```diff
@@ -1,13 +1,43 @@
 package com.toptier.targetmvd.ui.views;
 
+import android.widget.ArrayAdapter;
+import android.widget.Spinner;
+
 import com.squareup.otto.Bus;
+import com.toptier.targetmvd.R;
 import com.toptier.targetmvd.ui.activities.SignUpActivity;
 
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
 public class SignUpView extends BaseView {
     private Bus mBus;
 
+    @Bind(R.id.spinner_gender) Spinner genderSpinner;
+
     public SignUpView(SignUpActivity activity, Bus bus) {
         super(activity);
         this.mBus = bus;
+
+        ButterKnife.bind(this, activity);
+
+        setActionBar(getActivity(), getActivity().getString(R.string.sign_up));
```

Comment:
Here you can use `setActionBar` located on `BaseView` which also enables you the back option.